examples/locality-load-balancing/docker-compose.yaml

```diff
@@ -0,0 +1,55 @@
+version: "3.7"
+services:
+  client-envoy:
+    build:
+      context: .
+      dockerfile: Dockerfile-client
+    hostname: client
+    ports:
+      - 8001:8001
+    networks:
+      - envoymesh
+    depends_on:
+    - "backend-local-1"
+    - "backend-local-2"
+    - "backend-remote-1"
+    - "backend-remote-2"
+  backend-local-1:
+    build:
+      context: .
+      dockerfile: Dockerfile-server
+    environment:
```

Comment:
i think the `host` and `environment` lines are not required as the service has this name - same with services defined below

Reply:
`host` can be remove. Environment variable is required for the Python service to work.